python/samples/agentchat_graphrag/settings.yaml

```diff
@@ -0,0 +1,130 @@
+### This config file contains required core defaults that must be set, along with a handful of common optional settings.
+### For a full list of available settings, see https://microsoft.github.io/graphrag/config/yaml/
+
+### LLM settings ###
+## There are a number of settings to tune the threading and token limits for LLM calls - check the docs.
+
+encoding_model: cl100k_base # this needs to be matched to your model!
+
+llm:
+  api_key: null
+  type: azure_openai_chat # or azure_openai_chat
+  model: gpt-4o
+  model_supports_json: true # recommended if this is available for your model.
+  audience: "https://cognitiveservices.azure.com/.default"
+  api_base: https://<resource-name>.openai.azure.com
+  api_version: 2024-08-01-preview
+  deployment_name: gpt-4o
+
+parallelization:
+  stagger: 0.3
+  # num_threads: 50
+
+async_mode: threaded # or asyncio
+
+embeddings:
+  async_mode: threaded # or asyncio
+  vector_store:
+    type: lancedb
+    db_uri: 'data\output\lancedb'
```

Comment:
This is using Windows backslash. Should make a note here as well as in the README.md